cv19ResSupportV3/V3/Boundary/Requests/HelpRequestPatchRequest.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+
+namespace cv19ResSupportV3.V3.Boundary.Requests
+{
+    public class HelpRequestPatchRequest
```

Comment:
[nothing to do here, I'm just ranting] 


Arguably the difference between a PATCH and PUT is that the PATCH will only update fields that it receives, therefore any field within the PATCH should be optional, whereas the PUT we'd expect to have some serverside validation.

I see that this PATCH and the PUT (Update) objects look different, I guess that's because the current frontend uses them for different purposes.. We may wish to align that as we move forward with the new api